### PR TITLE
Enable copy/paste of commands

### LINKED ISSUE #666: Update samples to enable copy/paste
## Summary

Update the samples to enable the user to copy/paste the samples. 

### Basic example

Ask the user to provide some values to environment variables, then all the subsequent commands can be pasted directly

### Motivation

Easing the entry bar for users to get started.

### PR DESCRIPTION
**Pull request by SteveLasker**
Resolves #666 

Replaces: #654 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by SteveLasker** [comment]
cc: @beekman33

**Comment by SteveLasker** [comment]
These changes are ready to merge, covering:
- [Getting Access Tokens using a Custom Integration](https://docs.rkvst.com/developers/developer-patterns/getting-access-tokens-using-app-registrations/)
- [Containers as Assets](https://docs.rkvst.com/developers/developer-patterns/containers-as-assets/)

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md [hunk 1]
```diff
@@ -16,207 +16,255 @@ aliases:
...
+  {{< img src="IntegrationsTab.png" alt="Rectangle" caption="<em>Navigate to Settings, then Integration</em>" class="border-0" >}}
+1. Click the `Custom` box to create a Custom Integration
+1. Enter any `Display Name` you'd like
+  {{< img src="Confirm.png" alt="Rectangle" caption="<em>Completed Web Registration</em>" class="border-0" >}}
+  {{< note >}}
+  Optionally add any `Custom claims` at this step by clicking the `+ Add` button. Ensure the `Name` _does not start_ with `jit_` (RKVST reserved names) or use any other [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims).
+  {{< /note >}}  
+1. Once complete, click `Confirm` to complete the custom integration
+1. You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint
+    {{< caution >}}
```
Comment (pcnorth) [inline content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:49]:
```suggestion
{{< caution >}}
```

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md [hunk 2]
```diff
@@ -16,207 +16,255 @@ aliases:
...
+    **Caution:** Save the `CLIENT_ID` and `SECRET` to a password manager or secret management service as the `SECRET` can **not** be viewed again. A new `SECRET` can be regenerated by editing the Integration, invalidating the previous `SECRET`.
```
Comment (pcnorth) [inline content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:50]:
```suggestion
**Caution:** Save the `CLIENT_ID` and `SECRET` to a password manager or secret management service as the `SECRET` can **not** be viewed again. A new `SECRET` can be regenerated by editing the Integration, invalidating the previous `SECRET`.
```

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md [hunk 3]
```diff
@@ -16,207 +16,255 @@ aliases:
...
+    {{< /caution >}}
```
Comment (pcnorth) [inline content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:51]:
```suggestion
{{< /caution >}}  
```

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md [hunk 4]
```diff
@@ -16,207 +16,255 @@ aliases:
...
+  {{< img src="Confirm.png" alt="Rectangle" caption="<em>Completed Web Registration</em>" class="border-0" >}}
+  {{< note >}}
+  Optionally add any `Custom claims` at this step by clicking the `+ Add` button. Ensure the `Name` _does not start_ with `jit_` (RKVST reserved names) or use any other [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims).
+  {{< /note >}}  
+1. Once complete, click `Confirm` to complete the custom integration
+1. You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint
+    {{< caution >}}
+    **Caution:** Save the `CLIENT_ID` and `SECRET` to a password manager or secret management service as the `SECRET` can **not** be viewed again. A new `SECRET` can be regenerated by editing the Integration, invalidating the previous `SECRET`.
+    {{< /caution >}}  
+    {{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
```
Comment (pcnorth) [inline content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:52]:
```suggestion
{{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
```

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md (hunk 1 above)
Comment (pcnorth) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:49]: These spaces prevent the Caution box from displaying correctly.

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md [hunk 5]
```diff
@@ -16,207 +16,255 @@ aliases:
...
-**Note:** By default, newly created Integrations will always have a Non-Administrator permission to the API, you must add the Integration as an Administrator to elevate it's permissions.
-
-You can add a Custom Integration as an Administrator using the `Settings` screen, where the issuer will be `https://app.rkvst.io/appidpv1` and the subject will be your Integration's `CLIENT_ID`.
+**Note:** Creating App Registrations requires **Administrator** privileges.  
+If `Settings` does not appear in the navigation, see your RKVST Administrator for access.
 {{< /note >}}
 
-### Using the App Registrations API to create an Integration
+### Using the RKVST App to Create an Integration (First-Time Setup)
+
```
Comment (pcnorth) [inline content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:36]: Is there a reason why these bullet points are all 1. ? I still see a numbered sequence in the browser.

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md [hunk 6]
```diff
@@ -16,207 +16,255 @@ aliases:
...
+  Optionally add any `Custom claims` at this step by clicking the `+ Add` button. Ensure the `Name` _does not start_ with `jit_` (RKVST reserved names) or use any other [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims).
+  {{< /note >}}  
+1. Once complete, click `Confirm` to complete the custom integration
+1. You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint
+    {{< caution >}}
+    **Caution:** Save the `CLIENT_ID` and `SECRET` to a password manager or secret management service as the `SECRET` can **not** be viewed again. A new `SECRET` can be regenerated by editing the Integration, invalidating the previous `SECRET`.
+    {{< /caution >}}  
+    {{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
+2. Copy the `CLIENT_ID` and `SECRET` to a password manager for future reference
+3. Skip to [Grant Permissions to Custom Integration](#grant-permissions-to-custom-integration) to configure an Access Policy
```
Comment (pcnorth) [inline content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:54]: Jon suggested deleting the section "Using the App Registrations API to Create an Integration". When this is done there is no need to skip to the section.
```suggestion
9. The next section describes how to configure an Access Policy for the Integration
```

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md [hunk 7]
```diff
@@ -16,207 +16,255 @@ aliases:
...
+
+### Using the App Registrations API to Create an Integration
```
Comment (pcnorth) [inline content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:56]: I agree with Jon that this section should be deleted. I've never used it and the article flows much better without the interruption.

**Review by pcnorth — CHANGES_REQUESTED** [review]
I found some indentation issues that stopped a Caution box from working. I also agree with Jon's comments in Slack about deleting a section.
Also, the bullet point numbering is out of sequence but it still displays ok when I run it locally and view it in my browser.

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md (hunk 5 above)
Comment (pcnorth) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:36]: I've just found the reason while reviewing the style guide changes :-)

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md (hunk 1 above)
Comment (SteveLasker) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:49]: @pcnorth, curious why you're seeing this differently. Here's what I see locally:
image

What's also interesting: if I undent the block from lines 49-52, the ordered list still maintains the number counting. 🤔

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md (hunk 2 above)
Comment (SteveLasker) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:50]: see above

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md (hunk 3 above)
Comment (SteveLasker) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:51]: see above

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md (hunk 4 above)
Comment (SteveLasker) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md]: see above

**Comment by SteveLasker** [comment]
I've addressed all the feedback, thank you, and added markdown linting fixes.

File: content/developers/developer-patterns/containers-as-assets/index.md [hunk 8]
```diff
@@ -104,38 +124,61 @@ steps:
...
+
+If errors occur, see [Troubleshooting Token Generation](../getting-access-tokens-using-app-registrations/#troubleshooting-token-generation)
 {{< /tab >}}}
 {{< /tabs >}}
 
-It is now recorded that there is a `Box` within the container `Shipping Container`. We repeat this process to create another Asset and record what was inside the `Box`.
+The `Box(es)` have been recorded as being within the `Shipping Container`.
 
-## List All Assets Asssociated with a Container
+## List All Assets Associated With a Container
```
Comment (mmwilder26) [inline content/developers/developer-patterns/containers-as-assets/index.md:169]: Was the capital "W" in "W"ith on purpose?

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md [hunk 9]
```diff
@@ -16,207 +16,215 @@ aliases:
...
-3. Submit your new Application JSON to the App Registration API endpoint. 
+To test the creation of the Custom integration and the configuration of the bearer token file (`rkvst-bearer.txt`), query the assets API
 
 ```bash
-curl -X POST \
-     -H "@$BEARER_TOKEN_FILE" \
-     -H "Content-Type: application/json" \
-     -d "@/path/to/jsonfile" \
-     https://app.rkvst.io/archivist/iam/v1/applications
+curl -H "@rkvst-bearer.txt" \
```
Comment (mmwilder26) [inline content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:133]: might be an idea to explicitly add the GET as to make it clear and it aligns with the current API documentation

@SteveLasker

**Review by pcnorth — APPROVED** [review]

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md (hunk 9 above)
Comment (pcnorth) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:133]: Good spot, people who are new to curl or REST might need the examples to be free of assumptions.

**Comment by pcnorth** [comment]
I've checked the latest version locally and the things that I raised are resolved. I would recommend including Madia's suggestion too.

File: content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md (hunk 9 above)
Comment (SteveLasker) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:133]: Thanks @mmwilder26, it was the warning from the `curl` command that GET is the default that caused me to remove it. 
I do like the explicit command.

Comment (SteveLasker) [reply content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md:133]: I just realized, the text: 
> Unnecessary use of -X or --request, GET is already inferred.

Is only present when `-v` is also passed.
```bash
curl -v -X GET -H "@rkvst-bearer.txt"     https://app.rkvst.io/archivist/v2/assets | jq
Note: Unnecessary use of -X or --request, GET is already inferred.
```

I've updated to include `-X GET` and remove `-v` as it was overly noisy for the getting started experience.

**Review by mmwilder26 — APPROVED** [review]

**Pull request merged**